nextflow_schema.json

```
@@ -96,6 +96,22 @@
           "format": "file-path",
           "mimetype": "text/tab-separated-values",
           "description": "Table of marker genes used to validate consensus cell types"
+        },
+        "cell_type_ewings_auc_max_rank": {
+          "type": "integer",
+          "default": 425
+        },
+        "cell_type_ewings_msigdb_list": {
+          "type": "string",
+          "default": "https://raw.githubusercontent.com/AlexsLemonade/OpenScPCA-analysis/refs/tags/v0.2.2/analyses/cell-type-ewings/references/msigdb-gene-sets.tsv"
```

Can you add all of the validataion info here and for the other files? I made an attempt at the description for this one, but please update it as needed.. A description for the `max_rank` would also be useful.

```suggestion
          "default": "https://raw.githubusercontent.com/AlexsLemonade/OpenScPCA-analysis/refs/tags/v0.2.2/analyses/cell-type-ewings/references/msigdb-gene-sets.tsv"
                    "pattern": "\\.tsv$",
          "format": "file-path",
          "mimetype": "text/tab-separated-values",
          "description": "Table of MSigDB gene sets"
```